toolkit/components/console/jsconsole-clhandler.manifest

```diff
@@ -1,3 +1,13 @@
-component {2cd0c310-e127-44d0-88fc-4435c9ab4d4b} jsconsole-clhandler.js
-contract @mozilla.org/toolkit/console-clh;1 {2cd0c310-e127-44d0-88fc-4435c9ab4d4b}
+# Disable jsconsole-clhandler so it doesn't override devtools-startup.
+# Firefox depends on devtools-startup overriding jsconsole-clhandler because
+# it puts it into the components/browser/ subdirectory, which is apparently
+# registered after the top-level components directory.  But Positron puts
+# devtools-startup into the top-level directory, so jsconsole-clhandler
+# overrides it.  This hack prevents it from doing so.
+#
+# XXX Figure out a better way to prevent jsconsole-clhandler from overriding
+# devtools-startup.
+#
+# component {2cd0c310-e127-44d0-88fc-4435c9ab4d4b} jsconsole-clhandler.js
+# contract @mozilla.org/toolkit/console-clh;1 {2cd0c310-e127-44d0-88fc-4435c9ab4d4b}
 category command-line-handler b-jsconsole @mozilla.org/toolkit/console-clh;1
```

Comment:
Let's revert this back to what it was, except change it's entry so it runs later. Looking across the current [landscape][1] of handler entries, maybe something like `t-jsconsole` (t for toolkit), which would be near the bottom, but still before things like `x-*` and `y-*` which are trying be default / final handlers.

Then let's add a new category entry for `devtools-startup.js` (with its own contract ID as well) which uses the "normal" priority "m", something like `m-devtools`.

This should give the right effect for Positron and maintain it for Firefox: `devtools-startup` runs before the toolkit handler. It does mean that `--jsconsole` appears twice in the help output, but that seems acceptable I think.

I wanted to test my theories, so I opened #42 for this part.

[1]: https://dxr.mozilla.org/mozilla-central/search?q=command-line-handler&redirect=false&case=true